### DISCUSSION TITLE
the deformation result

### DISCUSSION THREAD

**Discussion by xander9797** [gmtsar/gmtsar#947] [Q&A]
The deformation result I produced using the SAR images of Sentinel-1a satellite contained large grid pixel sizes. Is this an issue with my parameter selection or with the image resolution？

**Comment by Xiaohua-Eric-Xu** [comment]
There are two steps that may control your final resolution. 1) decimation factor during interferometry step (i.e., range_dec and azimuth_dec + blurry from filter_wavelength); 2) geocoding resolution that is dependent on filter_wavelength, but you could change this at the step of geocoding when you run proj_ra2ll.csh (final optional parameter).

**Reply by xander9797** [reply]
thankyou for your answer
